Classes/Bookmark/BookmarkStore.swift

```diff
@@ -75,7 +75,7 @@ final class BookmarksStore {
 
     var bookmarks: [BookmarkModel] {
         refresh()
-        return Array(_bookmarks)
+        return Array(_bookmarks.sorted(by: { "\($0.owner)\($0.name)\($0.number)" > "\($1.owner)\($1.name)\($1.number)"}))
```

Comment:
What about making `BookMarkModel` conform to `Comparable` and using `>`?

Reply:
Also is sorting the array _every_ time we fetch this value the best idea? I wonder if there's a better place we could do this?